website/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -507,7 +507,7 @@ This endpoint returns a list of key names at the specified location. Folders are
 suffixed with `/`. The input must be a folder; list on a file will not return a
 value. Note that no policy-based filtering is performed on keys; do not encode
 sensitive information in key names. The values themselves are not accessible via
-this command.
+this command. The token used to list KV secrets must contain `metadata` in it's policy.
```

Comment:
I think the wording here could stand to be improved a bit - I see a risk it will not communicate clearly to readers:

* Not all policies are token policies - granting the access via an identity policy would also be a valid way to allow listing.
* To say the policy must contain the string `metadata` is a bit imprecise. Really, it must contain a path stanza granting `list` capability on the metadata path.
* In context, `it's` is not the right grammar.

What about something along the lines of:

To list secrets in a KV v2, a user must have a policy granting them the `list` capability on this `/metadata/` path - even all the rest of their interactions with the KV v2 are via the `/data/` APIs, access to at least `list` the `/metadata/` path should typically also be granted.

?